plugins/analogix-usbc/fu-plugin-analogix-usbc.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2021 Xiaotian Cui <xtcui@analogixsemi.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include "fu-plugin-vfuncs.h"
+#include "fu-hash.h"
+
+#include "fu-analogix-usbc-device.h"
+#include "fu-analogix-usbc-firmware.h"
+
+#define MINIBONS_PARENT_GUID                "cfc5f783-2f3c-5db0-9d09-d5a3044eabd9"

```

Comment:
This parent guid would you be able to leave in a comment above the string used to build it?